packages/medusa/src/api/admin/views/[entity]/configurations/[id]/route.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "@medusajs/framework/http"
+import { AdminUpdateViewConfigurationType } from "../validators"
+import { HttpTypes, ISettingsModuleService } from "@medusajs/framework/types"
+import { MedusaError, Modules } from "@medusajs/framework/utils"
+
+export const GET = async (
+  req: AuthenticatedMedusaRequest<HttpTypes.AdminGetViewConfigurationParams>,
+  res: MedusaResponse<HttpTypes.AdminViewConfigurationResponse>
+) => {
+  const settingsService: ISettingsModuleService = req.scope.resolve(
```

Comment:
Could you add `ISettingsModuleService` type here:
https://github.com/medusajs/medusa/blob/develop/packages/core/framework/src/types/container.ts#L38

Then you can remove `: ISettingsModuleService` everywhere